htdocs/projet/element.php

```diff
@@ -1259,6 +1259,10 @@
 					if (!empty($element->ref_customer)) {
 						print ' - '.$element->ref_customer;
 					}
+					// Compatibility propale
+					if (empty($element->ref_customer) && !empty($element->ref_client)){
+						print ' - '.$element->ref_client;
+					}
```

Comment:
@ibuiv 

if (!empty($element->ref_customer) || !empty($element->ref_client)) {
	print ' - '.(!empty($element->ref_customer) ? $element->ref_customer : $element->ref_client);
}

Reply:
@ibuiv you can just test with 3 lines !